collector/index.md

```diff
@@ -0,0 +1,498 @@
+---
+title: 'pganalyze Collector'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The pganalyze collector periodically queries your database and sends metrics and metadata (as "snapshots") to
+the pganalyze app. For details on setting up the collector, see the [collector installation guide](/docs/install/02_installing_the_collector).
+The collector is open-source and the code is available [on GitHub](https://github.com/pganalyze/collector).
+
+```toc
+```
+
+# Security
+
+Keeping your data safe is very important to us. We take a number of precautions to minimize the data
+we have access to, and to limit our level of access to that data. The collector is the only pganalyze
+component that talks directly to your database. In our recommended configuration, the collector uses
+a database role with limited access, and can only read system statistics and metadata&mdash;it cannot actually
+query your data.
+
+The collector connects to your database on the port you specify (default 5432) using the standard Postgres
+protocol, to Amazon S3 also over https (port 443) to upload monitoring data, and to the pganalyze API also
+over https. The collector only makes outbound connections&mdash;it does not listen on any ports.
```

Comment:
That's great, thanks. I'm rewording the Heroku section slightly because the collector app is acting as the drain in this case (the drain in Heroku terminology is what consumes the logs, not the mechanism to get them there). But I think it's great how explicit we can be here.